lib/android/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     exclude group: 'com.android.support'
   }
   implementation "com.android.support:appcompat-v7:${safeExtGet('supportLibVersion', '28.0.0')}"
+  implementation "com.google.android.gms:play-services-location:16.0.0"
```

Comment:
I guess this should be using the same `${safeExtGet('playServicesVersion', '16.1.0')` mechanism?

Reply:
Actually I discovered that play-services-location does not exactly follow the same versions as the rest of Play Services.

For example, play-services-location does not have a version 16.1.0:
https://mvnrepository.com/artifact/com.google.android.gms/play-services-maps

But for example play-services-maps does:
https://mvnrepository.com/artifact/com.google.android.gms/play-services-maps

Or am I missing something? If not, there are two ways to do it:

1) Use a constant hard coded version like I did.
2) Expose another Gradle variable like "locationServicesVersion" or something like that.

Reply:
You're right. I looked directly in the [Google maven repo](https://dl.google.com/dl/android/maven2/index.html) too and it doesnt seem to have version 16.1.0 for `play-services-location`. That's confusing!
Thanks for explaining, this seems reasonable then!